content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,328 @@
+and `version: v1` in namespace `foo`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: httpbin
+ namespace: foo
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.headers[version]
+     values: ["v1", "v2"]
+{{< /text >}}
+
+The policy allows principal `cluster.local/ns/default/sa/sleep` to access the workload
+using `GET` method when the request includes a `version` header of value `v1` or `v2`.
+Any requests not matched with the policy will be denied by default.
+
+That's all you need to enable access control for a workload, compared to the `v1alpha1`
+RBAC policy, you would need to configure three resources to achieve the same result:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    services: ["httpbin.foo.svc.cluster.local"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  rules:
+  - services: ["httpbin.foo.svc.cluster.local"]
+    methods: ["GET"]
+    constraints:
+    - key: request.headers[version]
+      values: ["v1", "v2"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  subjects:
+  - user: "cluster.local/ns/default/sa/sleep"
+  roleRef:
+    kind: ServiceRole
+    name: "httpbin"
+{{< /text >}}
+
+### Workload selector
+
+One major change in the `v1beta1` authorization policy is it now uses workload selector
+to specify where to apply the policy. This is the same workload selector used in
+the `Gateway`, `Sidecar` and `EnvoyFilter` configurations.
+
+The workload selector makes it clear that the policy is applied and enforced on workloads
+instead of services. If a policy applies to a workload that is used by multiple different
+services, the same policy will affect the traffic to all the different services.
+
+You can simply leave the `selector` empty to apply the policy to all workloads
+in a namespace. The following policy applies to all workloads in the namespace `bar`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: bar
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Root namespace
+
+A policy in the root namespace applies to all workloads in the mesh in every namespaces.
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`.
+
+For example, you installed Istio in `istio-system` namespace and deployed workloads in
+`default` and `bookinfo` namespace. The root namespace is configured to `istio-config`.
```

Comment:
Is this correct? by default the root namespace is istio-system.  or you mean user has purposely configured the root namespace to `istio-config`?

Reply:
if so, it is best to point to a link to show how to configure that.

Reply:
Yes, this is showing the example if the user changes the root namespace to `istio-config` from the default value. Updated the doc to make this more clear.

I think we can change the root namespace directly in the mesh config but I'm not sure how to do it through operator, which is the recommended way to install istio in 1.4. cc @ostromart